test/Benchmarks/src/Vector/Map_Error.enso

```diff
@@ -27,10 +32,10 @@ throw_ten_percent i = if i > 0 then i else
     Error.throw (Illegal_Argument.Error "error")
 
 collect_benches = Bench.build builder->
-    unaltered_data = create_vec
+    unaltered_data = Data.create
 
     type_group vector_type alter =
-        data = alter unaltered_data
+        data = alter unaltered_data.vec
```

Comment:
This still materializes the `unaltered_data.vec` eagerly here.

To make sure this is lazy, your `data` itself must be lazy too.

Reply:
Done